[2013-11-01 13:16:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif"" at C:\xampp\htdocs\DeveloppementDurable\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 116 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/framework/images/input_bg.gif\" at C:\\xampp\\htdocs\\DeveloppementDurable\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:116, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 13:16:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif"" at C:\xampp\htdocs\DeveloppementDurable\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 116 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/framework/images/input_bg.gif\" at C:\\xampp\\htdocs\\DeveloppementDurable\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:116, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 17:16:09] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 17:17:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 17:18:29] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 17:18:48] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/article=1"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/article=1\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 17:19:07] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/article=1"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/article=1\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 17:19:28] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/article=1"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/article=1\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 17:19:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/article=1"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/article=1\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-01 17:44:23] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/article=1"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/article=1\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:42:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:42:46] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:45:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:47:01] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:48:49] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:49:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:50:48] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:50:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 01:53:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/fondTR.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/fondTR.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 12:23:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/sc=2"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/sc=2\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 12:24:17] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/sc=2"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/sc=2\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-11 12:40:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/sc=1"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/sc=1\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:30)"} []
[2013-11-12 20:10:24] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /devdurable/so=10"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /devdurable/so=10\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:49)"} []
[2013-11-17 11:28:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /images/close-icone.png"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /images/close-icone.png\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:49)"} []
[2013-11-17 13:25:42] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1883 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1883, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:49)"} []
[2013-11-17 18:09:18] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:09:18] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:10:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:10:34] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:11:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:11:45] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:12:13] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:12:13] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:13:21] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:13:21] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:13:39] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:13:39] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:14:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:14:37] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:14:49] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:14:49] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:14:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:14:52] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:15:38] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:15:38] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:16:06] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:16:06] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:16:08] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:16:08] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:16:19] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:16:19] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:17:26] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:17:26] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:17:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:17:30] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:17:53] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:17:53] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:18:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:18:30] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:18:42] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:18:43] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:19:08] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:19:08] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:19:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:19:32] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:19:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:19:44] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:21:05] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:21:06] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:21:54] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:21:54] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:22:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:22:40] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:23:05] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:23:05] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:23:41] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:23:41] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:24:13] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:24:13] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:24:24] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/sky.jpg"" at C:\xampp\htdocs\DeveloppementDurable\app\cache\prod\classes.php line 1881 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
[2013-11-17 18:24:25] request.ERROR: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.") in "::base.html.twig" at line 49.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /css/sky.jpg\" at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\classes.php:1881, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\xampp\\htdocs\\DeveloppementDurable\\app\\cache\\prod\\appProdUrlMatcher.php:270)"} []
